tests/conftest.py

```
@@ -61,12 +61,38 @@ def client():
+    import asyncio
+    try:
+        from motor.motor_asyncio import AsyncIOMotorClient
+
+        async def _ping():
+            mongo_url = os.environ.get("MONGO_URL", "mongodb://localhost:27017")
+            client = AsyncIOMotorClient(mongo_url, serverSelectionTimeoutMS=2000)
+            await client.admin.command("ping")
+
+        asyncio.run(_ping())
+        return True
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)